tests/object.js

```diff
@@ -110,8 +106,21 @@ describe('Object', () => {
       }
     };
 
-    jsonexport(contacts, options, (err, csv)=>{
-      expect(csv).to.equal(`0,replaced-boolean${os.EOL}1,replaced-array${os.EOL}2,bad ace${os.EOL}a,parentless-a${os.EOL}b,replaced-string${os.EOL}c,replaced-number${os.EOL}d.x,other field`);
-    });
+    const csv = await jsonexport(contacts, options)
+    assert.equal(csv, `0,replaced-boolean${os.EOL}1,replaced-array${os.EOL}2,bad ace${os.EOL}a,parentless-a${os.EOL}b,replaced-string${os.EOL}c,replaced-number${os.EOL}d.x,other field`);
+  });
+
+  it('Zero, undefined, & null', async () => {
+    const csv = await jsonexport([
+      {
+        "a": 0,
+        "b": undefined,
+        "c": null,
+        "d": 1,
+        "e": "this"
+      }
+    ], {})
+
+    assert.equal(csv, `a,b,c,d,e\n0,,,1,this`)
```

Comment:
Moving the test here. Originally placed in the array.js tests as that test was added by the commit mentioned in the description. But it likely makes more sense here.

Reply:
Perfect!